src/openms/source/ANALYSIS/OPENSWATH/DIAHelper.cpp

```diff
@@ -267,60 +267,87 @@ namespace OpenMS
       double mass = product_mz;
       for (IsotopeDistribution::Iterator it = d.begin(); it != d.end(); ++it)
       {
-        isotopesSpec.push_back(std::make_pair(mass, it->getIntensity()));
+        isotopes_spec.push_back(std::make_pair(mass, it->getIntensity()));
         mass += mannmass;
```

Comment:
This should be mannmass / charge. But luckily (for charge = 1) it cancels out other bugs where always charge 4 was used. So only intensities were wrong.